app/code/Magento/Webapi/Model/Rest/Swagger/Generator.php

```diff
@@ -399,6 +404,14 @@ private function createQueryParam($name, $type, $description, $required = null)
         if (isset($required)) {
             $param['required'] = $required;
         }
+
+        // array types require collectionFormat = multi and '[]' appended to name, in order to properly send
+        // multiple values
+        if (!$this->getSimpleType($type)) {
```

Comment:
In this case ANY not simple type will get [], which also will include objects, so I assume we need exact type definition.